cordapp-cpk2/src/test/resources/cpk-export-reserved-package-no-auto-export/build.gradle

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id 'net.corda.plugins.cordapp-cpk2'
+}
+
+apply from: 'repositories.gradle'
+apply from: 'javaTarget.gradle'
+
+group = 'com.example'
+version = cordapp_version
+
+cordapp {
+    targetPlatformVersion = platform_version.toInteger()
+    minimumPlatformVersion = platform_version.toInteger()
+
+    contract {
+        name = 'cpk-export-reserved-package-no-auto-export'
+        versionId = cordapp_contract_version.toInteger()
+        licence = 'Test-Licence'
+        vendor = 'R3'
+    }
+}
+
+dependencies {
+    cordaProvided project(':corda-api')
+    implementation "commons-io:commons-io:$commons_io_version"
+}
+
+jar {
+    archiveBaseName = 'cpk-export-reserved-package-no-auto-export'
+    osgi {
+        autoExport = false
+    }
+}
```

Comment:
These two test projects are essentially the same, except one uses `autoExport = true` and the other `autoExport = false`. So why not unify them?
```gradle
jar {
    archiveBaseName = 'cpk-export-reserved-package`
    osgi {
        autoExport = osgiAutoExport
    }
}
```
Each test can pass its own value for the `osgiAutoExport` property via `buildAndFail()`:
```koltin
"-PosgiAutoExport=false"
```

Reply:
That's much better. Thank you.